### ISSUE TITLE
[EPIC] FFDec issue tracker

### ISSUE THREAD

**Issue by rayyaw** [rayyaw/flash-patcher#31]
Adding assets doesn't work correctly: https://www.free-decompiler.com/flash/issues/2102-adding-images-via-importimages-cli-is-not-working
Status: Pending reply and feature implementation by the ffdec team (Blocked)

**Comment by rayyaw** [comment]
This issue is blocked as we wait for the JPEXS devs to resolve it, not much to be done until we get a response.

**Comment by rayyaw** [comment]
JPEXS has indicated that we should use ffdec.bat instead of the exe, and that the new import is a known defect. We are looking into ways of fixing it but this issue will likely end up as a wontfix.

**Comment by rayyaw** [comment]
PR for Windows CLI fix: https://github.com/rayyaw/flash-patcher/pull/32

**Comment by rayyaw** [comment]
TODO: Create an issue. When JPEXS recompiles it should return an error code, allowing us to catch the issue.

**Comment by GTcreyon** [comment]
Avoiding #48 by fixing the root cause upstream:
https://www.free-decompiler.com/flash/issues/2129-stackoverflowerror-with-parallelspeedup-on-or-automaticdeobfuscation-off

**Comment by rayyaw** [comment]
Added to the top comment. :+1:

**Comment by rayyaw** [comment]
Stack overflow error when compiling with Parallel SpeedUp and no automatic deobfuscation: https://www.free-decompiler.com/flash/issues/2129-stackoverflowerror-with-parallelspeedup-on-or-automaticdeobfuscation-off
Status: wontfix. The user should manually use the fix of increasing the stack size themselves.
(Moving this out of the top comment so it only has open issues)

**Comment by rayyaw** [comment]
ffdec CLI is not working on GitHub Actions runners with the setup-java workflow: https://www.free-decompiler.com/flash/issues/2133-ffdec-sh-script-is-broken-on-some-java-distributions
Status: Pending + Demoted as this does not prevent compilation

**Comment by rayyaw** [comment]
ffdec CLI is not working on GitHub Actions runners with the setup-java workflow: https://www.free-decompiler.com/flash/issues/2133-ffdec-sh-script-is-broken-on-some-java-distributions
Status: Fix verified in latest nightly build. This is not a blocking issue.

**Comment by rayyaw** [comment]
We will begin work on importing new assets once the migration to pip is complete.

**Comment by rayyaw** [comment]
ffdec CLI not working on Windows: https://www.free-decompiler.com/flash/issues/2101-jpexs-cli-is-not-working-on-windows
Status: Fixed

**Comment by rayyaw** [comment]
Still no response to our latest comment. I will reach out to them again in the near future if they have not replied by then.

**Comment by rayyaw** [comment]
Since it looks like the FFDec team has no plans to fix our open issue I'm going to close this issue as "won't fix".
